src/factor.h

```diff
@@ -50,7 +52,7 @@ class CompactFactor {
    */
   CompactFactor(factor_id_t id);
 
-#define POTENTIAL(func_id) _potential_##func_id
+#define POTENTIAL_SIGN(func_id) _potential_sign_##func_id
```

Comment:
Not really... the `CF` one is still a misnomer... `potential` should have weight in it already.